src/Arduboy.h

```diff
@@ -41,17 +41,35 @@ class Arduboy : public Print, public ArduboyCore
    */
   boolean notPressed(uint8_t buttons);
 
-  /// Initializes the hardware
+  /// Initialize hardware, boot logo, boot utilities, etc.
   void begin();
-  /// Initializes the hardware (but with no boot logo)
-  void beginNoLogo();
+
+  /// Init just hardware, no logo, no boot utilities.
+  /**
+   * Look at the source for `begin()` and just rip out what you do not
+   * need and start there.  Calling just `boot()` might work also
+   * depending on your requirements.
+   *
+   * The minimum recommended `begin` replacement:
+   *
+   *   arduboy.boot()         // raw hardware init
+   *   arduboy.audio.begin()  // if you need audio
+   */
+  // void boot(); // defined in core.cpp
+
```

Comment:
You don't care but does anyone else, or has the idea been totally abandoned?

>To me it's useless if we aren't generating HTML and publishing it.

Is it not a good idea to publish library documentation somewhere on *arduboy.com*? I found that Doxygen did a good job for the [MicroView Libraray](http://learn.microview.io/doco/html/annotated.html), which is similar.